test/controllers/application_controller_test.rb

```diff
@@ -2,17 +2,16 @@
 
 class ApplicationControllerTest < ActionDispatch::IntegrationTest
   setup do
-    @user = users(:one)
-    @user_params = { 'user[email]' => @user.email, 'user[password]' => 'password' }
+    @user = users(:admin_one)
```

Comment:
Is there a reason to do `@user = users(:admin_one)` and then later `sign_in @user` instead of just `sign_in :admin_one` in one line?

If there is a reason to do it in two lines, perhaps we can re-name `@user` to `@user_admin` for clarify in the test.